### ISSUE TITLE
Error while running the Readme example

### ISSUE THREAD

**Issue by rishavs** [SuperPaintman/response-time#1]
for a src file with the code given in the README file;

```
require "response_time"
require "kemal"

add_handler ResponseTime::Handler.new

get "/" do |env|
  "Hello response time!"
end

Kemal.run
```

I am getting the following error:

```
Error in src/kemal.cr:1: while requiring "response_time"

require "response_time"
^

in lib/response_time/src/response_time.cr:6: HTTP::Handler is not a class, it's a module

  class Handler < HTTP::Handler
                  ^~~~~~~~~~~~~
```

Crystal version is 20.5 on the latest windows linux subsystem.

**Comment by SuperPaintman** [comment]
yeah, crystal's HTTP module api was changed
```crystal
  class Handler
    include HTTP::Handler
```

**Comment by SuperPaintman** [comment]
Please make PR with it

**Comment by rishavs** [comment]
sorry. I am just learning crystal. i just started my first hello world server on kemal today. 😀

**Comment by SuperPaintman** [comment]
ok, i closed this bug in 4f5e05e
